routes/index.js

```diff
@@ -1,9 +1,9 @@
 import express from 'express';
-import apiRouter  from './api';
+import apiRouter from './api';
 
 const router = express.Router();
 
 
-router.use("/api", apiRouter);
+router.use('/api', apiRouter);
```

Comment:
As requested by David, API version should be added here instead of adding them on individual routes to stay DRY `(Don't Repeat Your self)`